druid-lookups/src/main/java/com/yahoo/maha/maha_druid_lookups/query/lookup/namespace/ExtractionNamespace.java

```diff
@@ -4,16 +4,21 @@
 
 import com.fasterxml.jackson.annotation.JsonSubTypes;
 import com.fasterxml.jackson.annotation.JsonTypeInfo;
+import com.google.common.collect.ImmutableList;
 
 @JsonTypeInfo(use = JsonTypeInfo.Id.NAME, property = "type")
 @JsonSubTypes(value = {
         @JsonSubTypes.Type(name = "mahajdbc", value = JDBCExtractionNamespace.class),
-        @JsonSubTypes.Type(name = "mahainmemorydb", value = InMemoryDBExtractionNamespace.class)
+        @JsonSubTypes.Type(name = "maharocksdb", value = RocksDBExtractionNamespace.class),
```

Comment:
this rename is backward incompatible change.

Reply:
Yep, you are right, good catch.  I'll add a subtype for "mahainmemorydb".